client/css/style.css

```diff
@@ -220,12 +220,21 @@ kbd {
 #chat button.menu::before { content: "\f142"; /* http://fontawesome.io/icon/ellipsis-v/ */ }
 
 .context-menu-user::before { content: "\f007"; /* http://fontawesome.io/icon/user/ */ }
+
 .context-menu-chan::before { content: "\f0f6"; /* http://fontawesome.io/icon/file-text-o/ */ }
 .context-menu-close::before { content: "\f00d"; /* http://fontawesome.io/icon/times/ */ }
 
+.context-menu-action-whois::before { content: "\f29c"; /* http://fontawesome.io/icon/question-circle-o/ */ }
```

Comment:
Could you use http://fontawesome.io/icon/info-circle/ for this instead please? (`f05a`)